.changeset/@graphql-mesh_types-8716-dependencies.md

```
@@ -0,0 +1,7 @@
+---
+"@graphql-mesh/types": patch
+---
+dependencies updates:
+  - Added dependency [`@graphql-hive/pubsub@next` ↗︎](https://www.npmjs.com/package/@graphql-hive/pubsub/v/next) (to `dependencies`)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)